src/contracts/types/DAppApprovalTypes.sol

```diff
@@ -6,22 +6,22 @@ bytes32 constant DAPP_TYPEHASH = keccak256(
 );
 
 struct DAppOperation {
-    address from; // signor address
+    address from; // EOA of signer of the DAppOperation (SessionKey)
     address to; // Atlas address
-    uint256 nonce;
-    uint256 deadline;
-    address control; // control
-    address bundler; // msg.sender
+    uint256 nonce; // Atlas nonce of the DAppOperation available in the AtlasVerification contract
+    uint256 deadline; // block.number deadline for the DAppOperation
+    address control; // DAppControl address
+    address bundler; // Signer of the atlas tx (msg.sender)
     bytes32 userOpHash; // keccak256 of userOp.to, userOp.data
     bytes32 callChainHash; // keccak256 of the solvers' txs
-    bytes signature;
+    bytes signature; // DAppOperation signed by DAppOperation.from(SessionKey)
```

Comment:
Same as above. This is not necessarily a session key, it can also be a standard signatory.

Reply:
removed sessionKey label